### PR TITLE
Tag CodeTools.jl v0.4.5

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JunoLab/CodeTools.jl](https://github.com/JunoLab/CodeTools.jl)
Release: [v0.4.5](https://github.com/JunoLab/CodeTools.jl/releases/tag/v0.4.5)
Travis: [Travis Build Status](https://travis-ci.org/JunoLab/CodeTools.jl/builds/225104877)
Diff: [vs v0.4.3](https://github.com/JunoLab/CodeTools.jl/compare/bd2c187767ecd419dcb9e0b6365243d7f37317a5...757c504dffaffe9cb907203839158872af79537e)
`requires` vs v0.4.3: 
```diff
--- 0.4.3/requires
+++ 0.4.5/requires
@@ -2,3 +2,4 @@
 Lazy
 MacroTools
 LNR
+Tokenize
```
cc: @pfitzseb

### CODE REVIEW

File: CodeTools/versions/0.4.5/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.5-
```
Comment (tkelman) [inline CodeTools/versions/0.4.5/requires]: if you don't need to support prereleases, drop the trailing `-`

Comment (pfitzseb) [inline CodeTools/versions/0.4.5/requires]: Alright, should be find now (messed up the attobot release procedure, so sorry for the additional unnessecary update).

File: CodeTools/versions/0.4.5/requires [hunk 2]
```diff
@@ -0,0 +1,5 @@
+julia 0.5
+Lazy
```
Comment (tkelman) [inline CodeTools/versions/0.4.5/requires]: `@_` was introduced in Lazy 0.6 https://github.com/MikeInnes/Lazy.jl/commit/e4fedfc85db6604d667a0c2e2153723c1c116917

Comment (pfitzseb) [inline CodeTools/versions/0.4.5/requires]: True, thanks for noticing.

**Comment by pfitzseb** [comment]
Thanks for merging!

**Pull request merged**